src/nbl/video/IOpenGL_FunctionTable.h

```diff
@@ -1080,10 +1091,10 @@ namespace nbl {
 			{
 				// Save the previous bound vertex array
 				GLint restoreVertexArray;
-				_NBL_GL_CALL(m_logger, glGeneral.pglGetIntegerv(GL_VERTEX_ARRAY_BINDING, &restoreVertexArray));
+				//_NBL_GL_CALL(m_logger, glGeneral.pglGetIntegerv(GL_VERTEX_ARRAY_BINDING, &restoreVertexArray));
 				_NBL_GL_CALL(m_logger, glVertex.pglBindVertexArray(vaobj));
 				_NBL_GL_CALL(m_logger, glVertex.pglEnableVertexAttribArray(index));
-				_NBL_GL_CALL(m_logger, glVertex.pglBindVertexArray(restoreVertexArray));
+				//_NBL_GL_CALL(m_logger, glVertex.pglBindVertexArray(restoreVertexArray));
```

Comment:
@Crisspl WTF!?

Reply:
i guess i forgot to bring back, 
anyway this rebinding previous state is basically pointless, we dont let anyone call GL on our internal threads

Reply:
ive commented and will push it today anyway, but i dont think so.